pages/exhibition/_exhibition/_chapter.vue

```diff
@@ -0,0 +1,138 @@
+    <HeroImage
+      v-if="hero"
+      :image-url="heroImage.url"
+      :image-content-type="heroImage.contentType"
+      :header="page.name"
+      :lead="page.headline"
+      :rights-statement="hero.license"
+      :name="hero.name"
+      :provider="hero.provider"
+      :creator="hero.creator"
+      :url="hero.url"
+    />
+    <b-container>
+      <b-row>
+        <b-col
+          cols="9"
+          class="pb-3"
+        >
+          <h1
+            v-if="!hero"
+            data-qa="exhibition chapter title"
+          >
+            {{ page.name }}
+          </h1>
+          <article>
+            <h2>{{ page.description }}</h2>
+            {{ page.text }}
+          </article>
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col>
+          <BrowseSections
+            v-if="page"
+            :sections="page.hasPart"
+          />
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col>
+          <!-- TODO: add links to chapters, remove h2, move to component -->
+          <h2>Chapters</h2>
+          <ul v-if="chapters">
+            <li
+              v-for="chapter in chapters"
+              :key="chapter.fields.identifier"
+            >
+              <b-link :to="'/exhibition/' + exhibition + '/' + chapter.fields.identifier">
+                {{ chapter.fields.name }}
+              </b-link>
+            </li>
+          </ul>
+        </b-col>
+      </b-row>
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import createClient from '../../../plugins/contentful';
+  import BrowseSections from '../../../components/browse/BrowseSections';
+  import HeroImage from '../../../components/generic/HeroImage';
+
+  export default {
+    components: {
+      BrowseSections,
+      HeroImage
+    },
+    computed: {
+      hero() {
+        return this.page.primaryImageOfPage ? this.page.primaryImageOfPage.fields : null;
+      },
+      heroImage() {
+        return this.hero ? this.hero.image.fields.file : null;
+      }
+    },
+    asyncData({ params, query, error, app, store }) {
+      const contentfulClient = createClient(query.mode);
+      return contentfulClient.getEntries({
+        'locale': app.i18n.isoLocale(),
+        'content_type': 'exhibitionPage',
+        'fields.identifier': params.exhibition,
+        'include': 3,
+        'limit': 1
+      })
+        .then((response) => {
+          if (response.total !== 0 && response.items.total !== 0 && response.items[0].fields['hasPart'].total !== 0) {
+            const chapter = response.items[0].fields['hasPart'].find(c => c && c.fields.identifier === params.chapter);
+            if (chapter === undefined) {
+              error({ statusCode: 404, message: app.i18n.t('messages.notFound') });
+              return;
+            }
+            store.commit('breadcrumb/setBreadcrumbs', [
+              {
+                text:  app.i18n.t('exhibitions.exhibitions'),
+                href: '/exhibitions'
```

Comment:
Use `to` instead of `href` in all breadcrumbs so that the router is used.

Reply:
These URL paths are not localisation-aware. Use localePath, and named route objects instead.